src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -51,7 +51,7 @@
         "symfony/dependency-injection": "<4.4",
         "symfony/translation": "<4.4",
         "symfony/var-dumper": "<4.4",
-        "twig/twig": "<1.34|<2.4,>=2"
+        "twig/twig": ">=1|<2.4,>=2"
```

Comment:
this is wrong. It will conflict with all versions higher than 1 here. What you want is `<2` as conflict instead (which means you could change the rule to be only `<2.4`

Reply:
fixed